app/scripts/views/sign_up.js

```diff
@@ -225,6 +227,17 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       }
       return this._isMailcheckEnabledValue;
     },
+    _wasMailcheckUseful: function () {
+      var email = this.$el.find('.email');
+
+      if (email) {
+        var emailValue = email.val();
+        var mailcheckValue = email.data('mailcheckValue');
+        if (emailValue.length > 0 && mailcheckValue === emailValue) {
+          this.metrics.logEvent('tooltip.mailcheck-useful');
```

Comment:
There is already a `tooltip.mailcheck-used`, how is this different? Does this capture if the user types the email insted of clicks the tooltip?